ReadMe.md

```diff
@@ -2,22 +2,22 @@
 languages:
 - javascript
 page_type: sample
-description: "The sample implements an Outlook add-in that adds buttons to the Outlook ribbon."
+description: "The sample implements an Outlook add-in that accesses Microsoft Graph using single sign-on and adds buttons to the Outlook ribbon."
 products:
 - office
 - office-outlook
 urlFragment: outlook-ribbon-addin
 ---
 
-# AttachmentsDemo Sample Outlook Add-in
+# Single Sign-on (SSO) Sample Outlook Add-in
+
+The sample implements an Outlook add-in that uses Office's SSO feature to give the add-in access to Microsoft Graph data. Specifically, it enables the user to save all attachments to their OneDrive. It also shows how to add custom buttons to the Outlook ribbon. The sample illustrates the following concepts:
 
-The sample implements an Outlook add-in that adds buttons to the Outlook ribbon. It allows the user to save all attachments to their OneDrive. The sample illustrates the following concepts:
- 
-- Adding [add-in command buttons](https://docs.microsoft.com/outlook/add-ins/add-in-commands-for-outlook) to the Outlook ribbon when reading mail, including a UI-less button and a button that opens a task pane
-- Implementing a WebAPI to [retrieve attachments via a callback token and the Outlook REST API](https://dev.office.com/docs/add-ins/outlook/use-rest-api)
 - [Using the SSO access token](https://docs.microsoft.com/outlook/add-ins/authenticate-a-user-with-an-sso-token) to call the Microsoft Graph API without prompting the user
 - If the SSO token is not available, authenticating to the user's OneDrive using the OAuth2 implicit flow via the [office-js-helpers library](https://github.com/OfficeDev/office-js-helpers).
 - Using the [Microsoft Graph API](https://developer.microsoft.com/graph/docs/api-reference/v1.0/resources/onedrive) to create files in OneDrive.
+- Adding [add-in command buttons](https://docs.microsoft.com/outlook/add-ins/add-in-commands-for-outlook) to the Outlook ribbon when reading mail, including a UI-less button and a button that opens a task pane
+- Implementing a WebAPI to [retrieve attachments via a callback token and the Outlook REST API](https://dev.office.com/docs/add-ins/outlook/use-rest-api)
```

Comment:
@Rick-Kirkham Where does this link resolve to (since dev.office.com has been deprecated)?

Reply:
It should now go to https://review.docs.microsoft.com/en-us/office/dev/add-ins/outlook/implement-sso-in-outlook-add-in?branch=outlook-sso-sample. I'll fix it.

Reply:
You should update it to https://docs.microsoft.com/office/dev/add-ins/outlook/implement-sso-in-outlook-add-in

Reply:
Actually, now that I read the intro phrase carefully, it should go to get-attachments-of-an-outlook-item.md